classic/03 Configuring and Using CTI Adapter Features/11 Recording Controls.md

```diff
@@ -0,0 +1,9 @@
+<h2 class="toc">Recording Controls</h2>
+
+Recording Controls panel in the CCP Overlay allows your agents to control the recording behavior of the call.
+
+<img src="../media/ccp-overlay-5-recording.png" />
+
+This can be useful when you don't want to record every call, and give the agent the ability to pause and resume a recording.
+
+Note that once a recording is stopped, it cannot be restarted. After starting a recording, you should use pause/resume button to control it.
```

Comment:
Do we want to add the CTI Actions and recording controls documentation now if we're not releasing it?